app/providers/atmosphere/optimization_strategy/cpu_load.rb

```diff
@@ -0,0 +1,21 @@
+module Atmosphere
+  module OptimizationStrategy
+    class CpuLoad < OptimizationStrategy::Default
+      def monitor_vms?
+        true
+      end
+
+      def handle_event(hash)
```

Comment:
Unused method argument - `hash`. If it's necessary, use `_` or `_hash` as an argument name to indicate that it won't be used. You can also write as `handle_event(*)` if you want the method to accept any arguments but don't care about them.